java/fury-benchmark/src/main/java/io/fury/benchmark/Identity2IdMap.java

```diff
@@ -20,7 +20,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
-// based on fst Identity2IdMap, but with some jit inline optimization.
+// Drived from
+// https://github.com/RuedigerMoeller/fast-serialization/blob/e8da5591daa09452791dcd992ea4f83b20937be7/src/main/java/org/nustaq/serialization/util/FSTIdentity2IdMap.java.
```

Comment:
We modified it's implementation, to make it mor friendly for jit inline optimization.

Maybe we can reserve this information so new developers will know what we changed.